### PR TITLE
[5.6] Use in_array instead of duplicating condition

### PR DESCRIPTION
**Pull request by mnabialek**
Just small change that in addition will make adding more rules easier

### CODE REVIEW

File: src/Illuminate/Validation/ValidationRuleParser.php [hunk 1]
```diff
@@ -250,7 +250,7 @@ protected static function parseParameters($rule, $parameter)
     {
         $rule = strtolower($rule);
```
Comment (lucasmichot) [inline src/Illuminate/Validation/ValidationRuleParser.php]: or just `return preg_match('/(not(_)?)?regex/i', $rule) ? [$parameter] : str_getcsv($parameter);` ?

**Comment by browner12** [comment]
any performance concerns with the function call versus the straight up comparison operators?

**Comment by ntzm** [comment]
If any, it would be minute

**Comment by mnabialek** [comment]
@browner12 I don't think it would impact anything a lot. Assuming it won't be called thousands time per request I believe there won't be any visible difference

**Comment by browner12** [comment]
using PHPBench (thanks @ntzm) and the following tests:

**Original**

```php
class ValidatorTest
{
    /**
     * @Revs(1000)
     * @Iterations(5)
     */
    public function benchConsume()
    {
        $this->rule('');
        $this->rule('does not exist');
        $this->rule('regex');
        $this->rule('not_regex');
        $this->rule('notregex');
    }

    /**
     * @param string $rule
     * @return array
     */
    public function rule($rule)
    {
        if ($rule === 'regex' || $rule === 'not_regex' || $rule === 'notregex') {
            return [];
        }
    }
}
```

**New**

```php
class ValidatorNewTest
{
    /**
     * @Revs(1000)
     * @Iterations(5)
     */
    public function benchConsume()
    {
        $this->rule('');
        $this->rule('does not exist');
        $this->rule('regex');
        $this->rule('not_regex');
        $this->rule('notregex');
    }

    /**
     * @param string $rule
     * @return array
     */
    public function rule($rule)
    {
        if (in_array($rule, ['regex', 'not_regex', 'notregex'], true)) {
            return [];
        }
    }
}
```

I got the following results:

**Original**

screen shot 2018-06-13 at 10 19 36 am

**New**

screen shot 2018-06-13 at 10 19 44 am

So assuming I used the tool correctly, the new way is actually *slightly* faster.

**Pull request merged**